tpp-backend/desktop-image-experiment/desktop.md

```diff
@@ -0,0 +1,68 @@
+## Creating a Hyper-V image
```

Comment:
This directory `desktop-image-experiment` could probably have a better, less tentative-sounding name - I presume we think we're stuck with the manual route for the foreseeable future?

Reply:
As I understand it, we have a working automated setup based on a different base image, this is a separate not-quite-working manual setup based on the desktop base image. There could be an argument for removing this whole section from this PR & leaving it in a second open draft PR on the repo, with comments on what works/doesn't work.

Reply:
I will delete it.

There is one way to build the image that works, that's the tpp-backend/build-image.sh script. Starting from the desktop image was an experiment that was never completed, as I figured out how to do it via the server image.

Reply:
Given that it's been written, I think it's worth keeping /somewhere/, if not in the mainline.

Reply:
Ok, I will stick in a branch

It wasn't finished though. The .md file was me writing down what I was doing as I went along, in lieu of a script.